config/passport.js

```diff
@@ -0,0 +1,70 @@
+var passport=require('passport');
+
+var localStrategy=require('passport-local').Strategy;
+
+var dataModels = require('../database-mongo');
+
+passport.serializeUser(function(user,done){
+
+	done(null,user.id); //set user id inside session
+});
+
+passport.deserializeUser(function(id,done){
+
+	dataModels.User.findById(id,function(err,user){
+		done(err,user);
+	})
+});
+
+
+
+passport.use('local.signup',new localStrategy({
+	usernameField:'username',
+	passowrd:'password',
+	passReqToCallback:true,
+},function(req,username,password,done){
+	dataModels.User.findOne({'username':username},function(err,user){
+		if(err) return done(err);
+
+		if(user){
+
+			return done(null,false);
+
```

Comment:
Try and eliminate unnecessary spaces like these between lines. Otherwise, looks like you're using good indentation here. Maybe make the tab size a bit smaller, but other than that, good.